feathr_gradle/feathr-config/build.gradle

```diff
@@ -0,0 +1,88 @@
+apply plugin: 'java'
+apply plugin: 'pegasus'
+apply plugin: 'maven-publish'
+apply plugin: 'signing'
+
+dependencies {
+  implementation project(":feathr-data-models")
+  implementation project(path: ':feathr-data-models', configuration: 'dataTemplate')
+  implementation spec.product.avro
+  implementation spec.product.pegasus.data
+  implementation spec.product.typesafe_config
+  implementation spec.product.log4j
+  implementation spec.product.jsonSchemaVali
+  implementation spec.product.jackson.jackson_databind
+  implementation spec.product.mvel
+  implementation spec.product.json
+
+  testImplementation spec.product.testing
+  testImplementation spec.product.mockito
+  testImplementation spec.product.equalsverifier
+  testImplementation spec.product.mockito_inline
+}
+
+test {
+  maxParallelForks = 1
+  forkEvery = 1
+  // need to keep a lower heap size (TOOLS-296596)
+  minHeapSize = "512m"
+  useTestNG()
+}
+
+java {
```

Comment:
Do we still need to create a jar and publish for every single module here? Initially I wasn't able to create a single build directory and jar but I believe you were able to resolve that. In that case we should remove the publishing code in the gradle files in the individual modules and just have one for the overall project.